addons/discuss-button/userscript.js

```diff
@@ -11,7 +11,14 @@ export default async function ({ addon, global, console }) {
 
   if (addon.tab.clientVersion === "scratch-www") {
     while (true) {
-      const el = await addon.tab.waitForElement("div#navigation div.inner ul:not(.production)", { markAsSeen: true });
+      const el = await addon.tab.waitForElement("div#navigation div.inner ul:not(.production)", {
+        markAsSeen: true,
+        condition: () => {
+          if (!addon.tab.redux.state) return false;
+          if (!addon.tab.redux.state.scratchGui) return false;
+          return addon.tab.redux.state.scratchGui.mode.isPlayerOnly;
```

Comment:
You may use `?.` if wanted

Reply:
The code diff here has a logical error in it. Once fixed ?. probably won't work